controllers/v1/organizationExtension.js

```
@@ -0,0 +1,168 @@
+		return new Promise(async (resolve, reject) => {
+			try {
+				const orgExtension = await organizationExtensionHelper.createOrgExtension(req)
+				return resolve(orgExtension)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)